doc/api/esm.md

```diff
@@ -754,8 +754,9 @@ changes:
 * `specifier` {string}
 * `context` {Object}
   * `conditions` {string\[]} Export conditions of the relevant `package.json`
-  * `importAssertions` {Object}
-  * `parentURL` {string|undefined} The module importing this one, or undefined
+  * `importAssertions` {Object} The object after the `assert` in an `import`
+    statement, or the value of the `assert` property in the second argument of
```

Comment:
The "thing" after the `assert` keyword in static imports is technically not an object... I guess that's why it was never documented, it's just not easy to describe.

Reply:
It _is_ an object for `import()`. I don’t know what to call it for an `import` statement, but it looks like an object.

Reply:
Note that my comment was specifically about static imports.
Regarding dynamic imports, it's also incorrect, we do not pass the value, otherwise that would mean that the loader mutating the object would have an impact on the user code.

I've sent a suggestion in https://github.com/nodejs/node/pull/46153#discussion_r1066435761.